### ISSUE TITLE
WooCommerce Germanized 1.2 Fatal Error

### ISSUE THREAD

**Issue by maxxscho** [vendidero/woocommerce-germanized#5]
Hello,

I have the following setup.
- [WooCommerce 2.2.11](http://www.woothemes.com/woocommerce/)
- [The EventsCalendar 3.9.1](https://theeventscalendar.com/)
- [The EventsCalendar WooCommerce Tickets 3.9](https://theeventscalendar.com/product/wordpress-wootickets/)
- WooCommerce Germanized 1.1.1

Everything works fine.
Yesterday I updated to WooCommerce Germanized 1.2 and i get the following error
````
Fatal error: Call to a member function is_virtual_vat_exception() ...
````
where the I should be able to buy tickets on a single event.
![bildschirmfoto 2015-02-09 um 17 44 40](https://cloud.githubusercontent.com/assets/543690/6143755/286f4bb6-b1ce-11e4-978f-b03a26bc3b57.png)

If I downgrade to 1.1.1 all works well.

**Comment by vendidero** [comment]
Hi,
that is due to the fact, that the wootickets plugin doesn't use WooCommerce Product Factory to construct product objects - instead constructs objects "manually" - which is really bad practice..
Still I have just released (ba51db544b521412807e8b28c42e7541cb0fb9d5, latest commit) a fallback for those cases.. Will be implemented within next WP release.

Cheers

**Comment by maxxscho** [comment]
Ok, thanks! I'll give it a try!
